specification/storagesync/resource-manager/readme.md

```diff
@@ -26,6 +26,7 @@ These are the global settings for the Storage Sync API.
 
 ``` yaml
 openapi-type: arm
+tag: package-2018-07-01

```

Comment:
At this level, only one tag can be specified. It should continue to be the most recent API Version in `stable`

Reply:
Should I keep tag: package-2018-07-01 only ?